src/main/java/frc/trigon/robot/subsystems/arm/Arm.java

```diff
@@ -0,0 +1,122 @@
+package frc.trigon.robot.subsystems.arm;
+
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.controller.ElevatorFeedforward;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Arm extends SubsystemBase {
+    private final static Arm INSTANCE = new Arm();
+
+    private final CANSparkMax
+            masterAngleMotor = ArmConstants.MASTER_ANGLE_MOTOR,
+            followerAngleMotor = ArmConstants.FOLLOWER_ANGLE_MOTOR,
+            masterElevatorMotor = ArmConstants.MASTER_ELEVATOR_MOTOR,
+            followerElevatorMotor = ArmConstants.FOLLOWER_ELEVATOR_MOTOR;
+    private final TalonSRX elevatorEncoder = ArmConstants.ELEVATOR_ENCODER;
+    private final CANcoder angleEncoder = ArmConstants.ANGLE_ENCODER;
+
+    private final ArmFeedforward angleMotorFeedforward = ArmConstants.ANGLE_MOTOR_FEEDFORWARD;
+    private final ElevatorFeedforward elevatorMotorFeedforward = ArmConstants.ELEVATOR_MOTOR_FEEDFORWARD;
+
+    private TrapezoidProfile
+            angleMotorProfile = null,
+            elevatorMotorProfile = null;
+    private double
+            lastAngleProfileGeneration,
+            lastElevatorProfileGeneration;
+
+    public static Arm getInstance() {
+        return INSTANCE;
+    }
+
+    private double getAngleMotorPositionDegrees() {
+        return angleEncoder.getPosition().getValue();
+    }
+
+    private double getElevatorMotorPositionDegrees() {
+        return elevatorEncoder.getSelectedSensorPosition();
+    }
+
+    private double getAngleMotorVelocity() {
+        return angleEncoder.getVelocity().getValue();
+    }
+
+    private double getElevatorMotorVelocity() {
+        return elevatorEncoder.getSelectedSensorVelocity();
+    }
```

Comment:
Add the unit to the name and make this return in revolutions per second. See `Conversions`